sql/wsrep_sst.cc

```diff
@@ -661,7 +663,13 @@ static void *sst_joiner_thread(void *a) {
     /* Read committed isolation to avoid gap locking */
     thd->variables.transaction_isolation = ISO_READ_COMMITTED;
 
-    wsrep_sst_complete(thd, -err);
+    if (wsrep_sst_complete(thd, -err)) {
+      WSREP_ERROR(
+          "wsrep_sst_complete() failed, can't signal end of SST. "
```

Comment:
I am not sure if it is a good idea to put a function name in the error message. 

Can we use something like `Server encounterd a failure while signalling the completion of SST. Aborting` ? If we want to write the function name here, then we could consider `WSREP_DEBUG()` here.